tests/test-update-refspec-checkpoint.c

```diff
@@ -1493,6 +1493,10 @@ test_update_refspec_checkpoint_eos3a_eos4 (EosUpdaterFixture *fixture,
       { NULL, NULL, "Endless", "ELT-JWM", FALSE, NULL, NULL, NULL, TRUE, TRUE },
       { "os/eos/amd64/latest1", NULL, "Endless", "EE-200", FALSE, NULL, NULL, NULL, FALSE, FALSE },
       { "os/eos/amd64/latest1", NULL, "Endless", "EE-200", FALSE, NULL, NULL, NULL, TRUE, TRUE },
+      { "os/eos/amd64/latest1", NULL, "Standard", "EF20", FALSE, NULL, NULL, NULL, FALSE, FALSE },
+      { "os/eos/amd64/latest1", NULL, "Standard", "EF20", FALSE, NULL, NULL, NULL, TRUE, TRUE },
+      { "os/eos/amd64/latest1", NULL, "Standard", "EF20EA", FALSE, NULL, NULL, NULL, FALSE, FALSE },
+      { "os/eos/amd64/latest1", NULL, "Standard", "EF20EA", FALSE, NULL, NULL, NULL, TRUE, TRUE },
 
```

Comment:
Sorry if I'm missing something obvious, I only skimmed through this code, but here you are adding these systems in a function called `test_update_refspec_checkpoint_eos3a_eos4`. We actually want to allow these units to update to eos4.0, only not to eos5.0.

Reply:
Good question.

Reply:
Have a new function to handle the update from eos4 to eos5.